### ISSUE TITLE
machine learning beginner: how to get custom input data to train a conv net?

### ISSUE THREAD

**Issue by tu-lee** [tensorflow/tensorflow#2383]
I have started learning tensorflow recently. I am trying to input my custom python code as training data. I have generated random exponential signals and want the network to learn from that. This is the code I am using for generating signal-

`import matplotlib.pyplot as plt
import random
import numpy as np

lorange= 1
hirange= 10
amplitude= random.uniform(-10,10)
t= 10
random.seed()
tau=random.uniform(lorange,hirange)
x=np.arange(t)

plt.xlabel('t=time")
plt.ylabel('x(t)')
plt.plot(x, amplitude*np.exp(-x/tau))
plt.show()`

How can I use this graph as input vector in tensorflow?

**Comment by terrytangyuan** [comment]
Examples for tf.learn [here](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/examples/skflow) should make things easier for you.

**Comment by girving** [comment]
Questions like this should go on StackOverflow, not Github Issues.
